cli/options/options.go

```diff
@@ -73,6 +73,13 @@ var ConfigFile = cli.StringFlag{
 	Usage: "path to the node configuration file (overrides --config-path option)",
 }
 
+// RelativePath is a flag for commands that use node configuration and provides
+// the path to the consensus wallet instead of reading it from the config file.
+var RelativePath = cli.StringFlag{
+	Name:  "relative-path",
+	Usage: "path to the consensus wallet file (overrides path defined in the config)",
```

Comment:
s/`defined in the config`/`defined in the `Consensus` subsection of the node configuration file`